config/config.js

```diff
@@ -36,9 +37,39 @@ if (c.mongo.location[c.mongo.location.length-1] !== '/') {
 }
 
 // fs paths
-c.fs.base       = env.SUBSTITUTER_BASEPATH || '/tmp/o2r/';
+c.fs.base       = env.SUBSTITUTER_BASEPATH || '/tmp/o2r-dev/'; // '/tmp/o2r/'
 c.fs.compendium = c.fs.base + 'compendium/';
 
+// metadata extraction and brokering options
+c.meta = {};
+c.meta.doiPath = 'metadata.o2r.identifier.doi';
+
+// docker commands
+c.docker = {};
+
+c.bagtainer = {};
+c.bagtainer.docker = {};
+// See https://docs.docker.com/engine/reference/commandline/create/ and https://docs.docker.com/engine/reference/api/docker_remote_api_v1.24/#create-a-container
+c.bagtainer.docker.create_options = {
+  //AttachStderr: true,
+  //AttachStdin: false,
+  //AttachStdout: true,
+  //Cmd: ['bash', '-c', 'cat /etc/resolv.conf'],
+  CpuShares: 256,
+  //Cpuset: '',
+  //Domainname: '',
+  //Entrypoint: null,
+  Env: ['O2RPLATFORM=true'],
+  //Hostname: 'b9ea983254ef',
+  Memory: 1073741824, // 1G
+  MemorySwap: 2147483648, // double of 1G
+  NetworkMode: 'none',
+  Rm: true
+};
+// https://docs.docker.com/engine/reference/api/docker_remote_api_v1.24/#start-a-container
+c.bagtainer.docker.start_options = {
+};
+
 c.id_length = 6; // length of compendium ids [0-9,a-z,A-Z]
```

Comment:
change comment to `length of substitution ids`, because the loader only creates compendia with id length `5` (or whatever other length in the future).

Reply:
done